### ISSUE TITLE
Answer labels appear on next line

### ISSUE THREAD

**Issue by bh-j** [awsmug/torro-forms#347]
The answer options for all form questions appear on the next line (see attachments).

Question editor:

![screen shot 2017-06-30 at 11 37 10 am](https://user-images.githubusercontent.com/18666098/27743037-b46b361c-5d88-11e7-8721-b6f82bd36cfe.png)

Preview:

![screen shot 2017-06-30 at 11 32 57 am](https://user-images.githubusercontent.com/18666098/27743052-bed4d0ae-5d88-11e7-86e8-5fc1e05954d6.png)

**Comment by mahype** [comment]
Hi,

that looks like a CSS problem. Maybe the theme you are using occurs this display error. Are you able to check the CSS?

Greetings,

Sven

**Comment by bh-j** [comment]
Hi Sven,

Yes, it was CSS, thanks. I did experiment with CSS (although a weak and
unsuccessful effort) before sending the report, but got more focused today.

I successfully added the custom CSS below directly to the page containing
the form as I'm experimenting with WordPress and the free version disables
the Custom CSS feature.

Unfortunately, somewhere in today's efforts I picked up the following
run-time error which did not appear last week. It displays just above the
rendered form. I couldn't find the specified php file in Torro Forms'
plugin files. I'm sure it's something I did but would appreciate if you
could help.

I tried deactivating and reactivating Torro Forms and WordPress Native PHP
Sessions. I assume I'll lose my form should I delete and re-add the plugins
so I wanted to check with you on alternative solutions first.

Run-time error:

Warning: in_array() expects parameter 2 to be array, string given in
/srv/bindings/793cae097d574710b04d199fe475316a/code/wp-content/plugins/torro-forms/core/models/class-element-type.php
on line 158


Custom CSS:

<style>
.torro-form .torro-element input[type=checkbox],.torro-form .torro-element
input[type=radio] {
float:left;
margin:3px 10px 0px;
}
label {
font-weight: 600;
margin-bottom:6px;
}
legend {
font-weight: 600;
margin-bottom:6px;
}
</style>


Thanks!
-Brian
